build.cmd

```diff
@@ -213,6 +212,11 @@ echo generating clretwallmain.h and etmdummy.h
 "%PythonPath%"  "%genetw%" --man "%__SourceDir%\VM\ClrEtwAll.man" --exc "%__SourceDir%\VM\ClrEtwAllMeta.lst" --eventheader "%__GeneratedIntermediatesDir%\inc\ClrEtwAll.h" --macroheader "%__GeneratedIntermediatesDir%\inc\clretwallmain.h" --dummy "%__GeneratedIntermediatesDir%\inc\etmdummy.h" 
 IF  ERRORLEVEL 1 goto FailedToGenEtwMetadata 
 
+set "__GeneratedIntermediatesDirPresent=%__IntermediatesDir%\Generated"
```

Comment:
That can be done, but now for each and every utility function we would need to create a python file, The idea behind the inline function call is that eventually Utilities.py will contain other useful functions callable from shell. If it is better to create different files for different functions I will do it